### ISSUE TITLE
Feature Request - GUI Improvements for dates and playback info screen

### ISSUE THREAD

**Issue by cw-kid** [winsomehax/plugin.video.bitchute#25]
Hi 

It is possible to display the date of the videos somewhere? So you can tell if its a recent video or not? 

I am assuming the videos displayed in the lists are ordered from most recent at the top and get older as you navigate down the list and go to the next pages, is that correct? Still would be nice if the videos date is displayed on its Info screen or in the lists themselves. 

Whilst nothing is currently playing if you press the Info button on the remote control, on a video in the list you are taken to a page that looks like this. You can see a thumbnail image and the videos synopsis text is displayed. Using an Estuary Mod2 Skin. 

![image](https://user-images.githubusercontent.com/9383216/104369681-bee0a680-5515-11eb-8345-bea2b91b577f.png)

However whilst playback is in progress if you press Info it looks like this and the video title and synopsis is then not displayed, just the file name .mp4 is shown. 

![image](https://user-images.githubusercontent.com/9383216/104370031-30b8f000-5516-11eb-8391-40abf1d7e71d.png)

Thank you.

**Comment by winsomehax** [comment]
Yeah, they are in the default order of the bitchute screen they are scraped from - so most recent to oldest.

Adding the date to the info window shouldn't be hard - I'll take a look when I get some time. I burnt up the time I allocate to this stuff getting a trovo plugin up and running recently due to Dlive's behaviour... so it might be a week or so.

**Comment by winsomehax** [comment]
Testing now has date and duration in the infopane when scrolling through videos

**Comment by cw-kid** [comment]
Thanks ! 

When going in to the "Subscriptions" area and selecting a Channel I can now see the Date and Duration in the Info Pane for each video in the list.

However in other areas of the plugin like "Popular" "Trending" "Feed" for example, you don't see the Date and Duration. 

Many thanks

**Comment by winsomehax** [comment]
They've been added.
